.github/ISSUE_TEMPLATE/product-type-listing-request.md

```diff
@@ -8,14 +8,19 @@ assignees: ''
 
 ---
 
+## Instructions:
+1. Make a copy of the [New Product Listing Spreadsheet](https://docs.google.com/spreadsheets/d/1KugFl4qv9plCtFXfRgd5Ip0bXpAiZo6Yqm8CnWEBOUg/edit#gid=1512129648) and give the engineering team edit permissions
+2. Fill the copy New Product Listing Spreadsheet accordingly and link it to the **New Product Listing Spreadsheet** section below
+3. Fill the **Onchain data** and **Frontend** sections for each of the new product type
+4. Delete this **Instructions** section before submitting the Product Type Listing Request issue
+
 # Product Type Listing Request
 
-## Onchain data
-**Product Type Name**:
+**New Product Type Listing Spreadsheet**:
 
-**Product Type ID (only if updating an existing product type)**: N/A
+## Onchain data
 
-**Grace Period (in days)**:
+**Product Type Name**:
```

Comment:
Could we combine the `## Onchain data` and `## SDK data` in the issue template and change it to `## <productname>`? This way it is much easier to orentatie in the content of the issue while going back and forth while copy pasting data.